client/src/App.js

```diff
@@ -1,8 +1,10 @@
+import { Route, Switch } from 'react-router-dom';
 import { useState, useEffect } from 'react';
-// import StickyHeader from 'react-sticky-header';
-import axios from 'axios';
-import Coins from './Coins';
 import styled from 'styled-components';
+import Coins from './pages/Coins';
+import Portfolio from './pages/Portfolio';
+import Header from './Components/Header';
```

Comment:
components folder with small c 
```suggestion
import Header from './components/Header';
```

Reply:
Check! :)

Reply:
Done, thank you very much!